storages/sled-storage/src/store.rs

```diff
@@ -18,11 +18,7 @@ impl Store for SledStorage {
             State::Transaction {
                 txid, created_at, ..
             } => (txid, created_at),
-            State::Idle => {
-                return Err(Error::StorageMsg(
-                    "conflict - fetch_all_schemas failed, lock does not exist".to_owned(),
-                ));
-            }
+            State::Idle => lock::register(&self.tree, self.id_offset)?,
```

Comment:
I think this is good for now.
And it would be good for simplifying the codes in cli/lib.rs by following PR :)
ref. https://github.com/gluesql/gluesql/blob/6d147f54a9e91d7184748fc96a8d414b4395fffd/cli/src/lib.rs#L86